apps/sample-imdb/app/search/[term]/page.tsx

```diff
@@ -0,0 +1,21 @@
+import { Suspense } from 'react'
+import { SearchResult } from '~/app/search-result'
+import Loading from '~/app/loading'
+import { getTotalTitles } from '~/lib/db.server'
+import { HeaderNav } from '~/components/header-nav'
+
+export default async function Movie({ params }: { params: { term: string } }) {
+  const { term } = params
+  const aggregateTitles = getTotalTitles()
+  const { totalTitles = '0' } = await aggregateTitles
```

Comment:
Why split this`aggregateTitles`? 🤔

Reply:
not needed anymore. I had 2 requests and I wanted to make them in parallel. But now I moved the other one to another component.